src/core/lib/slice/slice_intern.cc

```diff
@@ -172,7 +172,8 @@ static const grpc_core::StaticMetadataSlice* MatchStaticSlice(
     const grpc_core::StaticMetadataSlice* static_slice_table =
         grpc_static_slice_table();
     if (ent.hash == hash && ent.idx < GRPC_STATIC_MDSTR_COUNT &&
-        static_slice_table[ent.idx].Equals(std::forward<SliceArgs>(args)...)) {
+        static_slice_table[ent.idx].operator==(
```

Comment:
The whole point of defining `operator==()` instead of `Equals()` is that you can just use the `==` operator here.  It should not be necessary to explicitly call `operator==()`.

Reply:
It breaks with std::forward() :(
Unless you know a way to get something like this working?
static_slice_table[ent.idx] == std::forward<SliceArg>(blah...)

Reply:
Done.